include/openenclave/host.h

```diff
@@ -384,6 +384,39 @@ void oe_free_key(
     uint8_t* key_info,
     size_t key_info_size);
 
+/**
+ * identity validation callback type
+ * @param[in] identity a pointer to an enclave's identity information
+ * @param[in] arg caller defined context
+ */
+typedef oe_result_t (
+    *oe_identity_verify_callback_t)(oe_identity_t* identity, void* arg);
+
+/**
+ * oe_verify_attestation_cert
+ *
+ * This function preform a custom validation on the input certificate. This
```

Comment:
Same comments in host.h as I added into enclave.h

Reply:
addressed